src/settings/NumberGeneratorSettings/constants.js

```diff
@@ -16,4 +16,6 @@ export const ACCESSION_NUMBER_SETTING = 'accessionNumber';
 export const BARCODE_SETTING = 'barcode';
 export const CALL_NUMBER_SETTING = 'callNumber';
 
+export const MOD_SETTINGS_API = 'settings/entries';
```

Comment:
Have permissions and interface been added for this endpoint?

Reply:
Here are the permissions for the settings:
https://github.com/folio-org/ui-inventory/blob/00722a01f949d702c2e6dada7430e41cd5e2b8be/package.json#L576C1-L596C11

And for `"permissionName": "ui-inventory.instance.view"` its already available:
https://github.com/folio-org/ui-inventory/blob/00722a01f949d702c2e6dada7430e41cd5e2b8be/package.json#L682C1-L683C75

Am I missing something else?